server/container_create.go

```diff
@@ -146,7 +146,13 @@ func resolveSymbolicLink(path string) (string, error) {
 
 func addDevices(sb *sandbox.Sandbox, containerConfig *pb.ContainerConfig, specgen *generate.Generator) error {
 	sp := specgen.Spec()
-	if containerConfig.GetLinux().GetSecurityContext().Privileged {
```

Comment:
just change this line to `if containerConfig.GetLinux().GetSecurityContext().GetPrivileged()` - the getter will ensure the code won't panic by still giving a default `Privileged` bool